lib/puppet-lint/plugins/check_whitespace/right_lbrace_whitespace.rb

```diff
@@ -0,0 +1,25 @@
+# Public: Check the manifest tokens for whitespace after the left bracket that is missing and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:right_lbrace_whitespace) do
```

Comment:
Again here, I can't think of a situation where it would be preferable to have no whitespace after the left bracket, except for {} or a newline.